assignments/prototypes.js

```diff
@@ -16,13 +16,58 @@
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
 
+const GameObject = function(attributes){
+  this.createdAt = attributes.createdAt;
+  this.name = attributes.name;
+  this.dimensions = attributes.dimensions;
+}
+
+GameObject.prototype.destroy = function(){
+  return `${this.name} was removed from the game.`
+}
+
+// const testObject = new GameObject({
+//   createdAt: 'December',
+//   name: 'Alice',
+//   dimensions: '34x32'
+// })
+
+// console.log(testObject);
+// console.log(testObject.destroy());
+
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
 
+function CharacterStats(charAttributes){
```

Comment:
I noticed that everyone was putting attributes and I was a little curious as to why, its cool to see you add some personalization to yours